BUILDING.md

```diff
@@ -594,6 +594,10 @@ rebuild may take a lot more time than previous builds. Additionally,
 ran `./configure` with non-default options (such as `--debug`), you will need
 to run it again before invoking `make -j4`.
 
+If you received the error `nodejs g++ fatal error compilation terminated cc1plus`
+during compilation, this is likely a memory issue and you should either provide
+more RAM or create swap space to accommodate toolchain requirements.
```

Comment:
```suggestion
If you received the error `nodejs g++ fatal error compilation terminated cc1plus`
during compilation, this is likely a memory issue and you should either provide
more RAM or create swap space to accommodate toolchain requirements or reduce
the number of parallel build tasks (`-j<n>`).
```

Reply:
FYI I'm running another local make job with -j1 to see if I can reproduce and will update. Probably going to take a few hours :-)

Reply:
If you are using less than 2 GB, even a single build task might run out of memory thanks to the nightmarish requirements of compiling (and linking!) C++ code. But setups with more available memory might still run into the same issue when utilizing parallel build tasks (e.g., `-j4` might run out of memory on an 8 GB system).

Reply:
Ahh makes sense. I'll commit your updates.